pwiz_tools/Shared/PanoramaClient/PanoramaUtil.cs

```diff
@@ -390,12 +391,21 @@ public static string Error(this FolderState state, Uri serverUri, string folderP
 
     public class PanoramaServerException : Exception
     {
+        public HttpStatusCode? HttpStatus { get; }
+
         public PanoramaServerException(string message) : base(message)
         {
         }
 
         public PanoramaServerException(string message, Exception e) : base(message, e)
         {
+            if (e is WebException webException)
+            {
+                using (var response = webException.Response as HttpWebResponse)

```

Comment:
This should be the last user of the WebException and WebResponse since we catch WebException and throw a PanoramaServerException.  So it seemed like the right place to call Dispose on the WebResponse. 
Where do you think it should go? 
The first answer (not the accepted one) in this SO post seems to suggest that it is not strictly necessary to dispose the WebResponse - https://stackoverflow.com/questions/32891627/should-webexception-response-getresponsestream-be-close-disposed

Reply:
I moved WebResponse disposal to a factory method in the PanoramaServerException class.   Does this look any better: https://github.com/ProteoWizard/pwiz/pull/2998/commits/90f824936ed5b5786e72a0da1823fb024b425215?